ios/fastlane/Fastfile

```diff
@@ -0,0 +1,26 @@
+# More documentation about how to customize your build
+# can be found here:
+# https://docs.fastlane.tools
+fastlane_version "1.109.0"
+
+# This value helps us track success metrics for Fastfiles
+# we automatically generate. Feel free to remove this line
+# once you get things running smoothly!
+generated_fastfile_id "7c72ded5-a420-4ff8-b16f-c9f89aa82349"
+
+default_platform :ios
+
+# Fastfile actions accept additional configuration, but
+# don't worry, fastlane will prompt you for required
+# info which you can add here later
+lane :beta do
+  match()
+  # build your iOS app
+  gym(
+    scheme: "Dora",
```

Comment:
are we still usign this name

Reply:
That's the Xcode project scheme name, which we can go through and change, but it doesn't show up anywhere other than in the build process.   And if we do change the name again (which is likely, because "In the unseen" isn't that great a name >___<), then we would have to change it again :P